bin/calyptos

```diff
@@ -66,10 +66,11 @@ def locate_cfgfile():
             # to return a valid config file name
             return item
 
-
 def run_driver(argp, operation, namespace=None, driver=None):
     namespace = namespace or argp.namespace or default_namespace
     driver = driver or argp.driver or default_driver
+    if operation == 'prepare' or operation == 'bootstrap' or operation == 'provision' or operation == 'uninstall':
+        cookbook_repo = ctx.obj['cookbook-repo']
```

Comment:
@mbacchi i believe that after @bigschwan's changes to use argparse the ctx variable is no longer used and you would have to pull this from the args